crates/cards/Cargo.toml

```
@@ -0,0 +1,14 @@
+[package]
+name = "cards"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+time = { version = "0.3.20", features = ["serde", "serde-well-known", "std"] }
+error-stack = "0.3.1"
+
+# First Party crates
+masking = { version = "0.1.0", path = "../masking" }
+common_utils = { version = "0.1.0", path = "../common_utils", features = ["signals", "async_ext"] }
```

```suggestion
common_utils = { version = "0.1.0", path = "../common_utils" }
```

I don't think we need either of the features here.